common/defs-flags.go

```diff
@@ -136,6 +136,21 @@ const (
 	FlagTaskQueueName           = "Name of the Task Queue."
 	FlagTaskQueueTypeDefinition = "Task Queue type [workflow|activity]"
 
+	// Namespace update flags
+	FlagActiveClusterDefinition           = "Active cluster name."
+	FlagClusterDefinition                 = "Cluster name."
+	FlagDescriptionDefinition             = "Namespace description."
+	FlagHistoryArchivalStateDefinition    = "History archival state, valid values are \"disabled\" and \"enabled\""
```

Comment:
should this and `FlagVisibilityArchivalStateDefinition` be boolean flags?

Reply:
in this case we may want to rename these as well similarly to cluster `--enable-connection` and `--tls-disable-host-verification` 

-- `enable-history-archival`
-- `enable-visibility-archival`

i'll start a thread separately from this PR

Reply:
can keep `visibility-archival-state` as well for backwards compatibility and make them invisible in --help

Reply:
upd. we should keep this as explicitly "enabled", "disabled" values as there is a 3rd option unspecified that afaik just takes the default according to settings